### PR TITLE
Add the "PingEvent" structure.

### PR DESCRIPTION
**Pull request by octo**
GitHub docs: https://developer.github.com/webhooks/#ping-event

### CODE REVIEW

File: github/event_types.go [hunk 1]
```diff
@@ -295,6 +295,18 @@ type PageBuildEvent struct {
 	Sender *User       `json:"sender,omitempty"`
 }
 
+// PingEvent is triggered when a WebHook is added to GitHub.
```
Comment (shurcooL) [inline github/event_types.go]: Minor, but for consistency with what I see above and below, call it "Webhook", not "WebHook".

File: github/event_types.go [hunk 2]
```diff
@@ -295,6 +295,18 @@ type PageBuildEvent struct {
...
+//
+// GitHub docs: https://developer.github.com/webhooks/#ping-event
+type PingEvent struct {
+	// Random string of GitHub zen.
+	Zen *string `json:"zen,omitempty"`
+	// The ID of the webhook that triggered the ping.
+	HookID *string `json:"hook_id,omitempty"`
+	// The webhook configuration.
+	Hood *Hook `json:"hook,omitempty"`
```
Comment (shurcooL) [inline github/event_types.go]: s/Hood/Hook.

File: github/event_types.go (hunk 1 above)
Comment (octo) [inline github/event_types.go]: Done.

File: github/event_types.go (hunk 2 above)
Comment (octo) [inline github/event_types.go]: Done.

**Comment by parkr** [comment]
Yay! I've been doing this [manually for a while](https://github.com/parkr/auto-reply/blob/e5c55feb1811aad9f2426fbe49891c50a3376e8f/hooks/event_types.go#L36-L38) and I appreciate that this may be added ❤️

**Comment by octo** [comment]
Thanks for the review @shurcooL!

Sorry for the repeated typos, I should probably go to bed ;) I fixed the commit message, not that it matters much if you typically squash commits when merging ;)

**Comment by shurcooL** [comment]
Friendly ping @gmlewis (no pun intended). Do you get notified when I [request a review](https://github.com/google/go-github/pull/499#event-902113237)?

**Comment by gmlewis** [comment]
Sorry for the delay, @shurcooL. Yes, I do get notified, but unfortunately if there are comments in the thread after that, the notification gets buried, so I missed it. Poor excuse, but the only one I have in this case. :smiley:
LGTM.
Feel free to merge, or I will later tonight.

**Comment by shurcooL** [comment]
> Sorry for the delay,

No problem! I was just checking. :) Thanks, I'll merge now.

Thank you @octo!

**Pull request merged**